tox.ini

```diff
@@ -8,13 +8,17 @@ allowlist_externals =
     coverage
     sphinx-build
     twine
+    gcloud
 # shared directory for re-used packages
 envdir = {toxinidir}/.env_tox
 passenv =
     CI
+    CLOUDSDK_*
     CONDA_PREFIX
     GITHUB_*
-    GOOGLE_APPLICATION_CREDENTIALS
+    GOOGLE_*
+    GCLOUD_*
+    GCP_*
```

Comment:
out of curiosity, which env vars ended up being the issue here?

Reply:
I didn't test them one at a time, I wound up mirroring what's in the chesire repository.